evidencecore/cryptolib/readkey_test.go

```diff
@@ -0,0 +1,32 @@
+package cryptolib
+
+import (
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func TestReadKey(t *testing.T) {
+	files, err := os.ReadDir("test-data")
+	assert.Nil(t, err)
```

Comment:
Please do it in all places
```suggestion
	assert.NoError(t, err)
```